source/_posts/2022-09-07-release-20229.markdown

```diff
@@ -0,0 +1,420 @@
+---
+layout: post
+title: "2022.9: Home Assistant Birthday Release!"
+description: "New revamped Automation Editor, Weekly Schedule helper to help with automations, and Paul Bottein (Mushroom Cards) joins Nabu Casa"
+date: 2022-09-07 02:00:00
+date_formatted: "September 7, 2022"
+author: Zack Barett
+author_twitter: zackbarettha
+comments: true
+categories:
+- Release-Notes
+- Core
+og_image: /images/blog/2022-09/social.png
+---
+
+<a href='/integrations/#version/2022.9'><img src='/images/blog/2022-09/social.png' style='border: 0;box-shadow: none;'></a>
+
+Home Assistant Core 2022.9! 🎂
+
+If you don't know yet, this month is Home Assistant's birthday! 🎉🎉 On September 17th, 2013 [Paulus Schoutsen](https://twitter.com/balloob) made the first commit for Home Assistant that started a snowball of Home Automation enthusiasts willing to contribute to the open-source project. 9 Years later, Home Assistant is actively used by more than 500,000 people and growing every day. Nabu Casa also turns 4 this month. Turning 4 years old and gaining a new full-time employee 🍄. This month it seems everyone was in the birthday spirit as we have a jam-packed release!
+
+The automation engine is a big reason why Home Assistant has so many users. So in the spirit of [Streamlining Experiences](https://www.home-assistant.io/blog/2022/01/19/streamlining-experiences/), we thought a revamp was due. You will notice some huge improvements to the look and feel of the Automation Editor that we think will greatly improve its usability. I know I for sure see a huge difference!
+
+Now I know that everyone usually expects [@frenck] to be writing these release notes, but he took a well deserved vacation! And now you are stuck with me. Zack 😀. I hope you enjoy the release and everything that was packed into it!
+
+-- Zack
+
+<!--more-->
+
+- [Paul Bottein joins Nabu Casa](#paul-bottein-joins-nabu-casa)
+- [Streamlining automations](#streamlining-automations)
+- [New helper: Weekly schedule](#new-helper-weekly-schedule)
+- [Bluetooth everywhere](#bluetooth-everywhere)
+- [Z-Wave Firmware updates now live](#z-wave-firmware-updates-now-live)
```

Comment:
```suggestion
- [Z-Wave JS Firmware Update Service now live](#z-wave-js-firmware-update-service-now-live)
```

Reply:
I don't think that is right, this is about providing updates in Home Assistant. Which we now target as "Z-Wave" not "Z-Wave JS".

Reply:
see comment below